tests/integration/bond_test.py

```diff
@@ -394,3 +394,26 @@ def test_create_vlan_over_a_bond(bond99_with_slave):
     ) as desired_state:
         assertlib.assert_state(desired_state)
     assertlib.assert_state(bond99_with_slave)
+
+
+@pytest.mark.xfail(
+    raises=NmstateVerificationError,
+    reason="https://bugzilla.redhat.com/1789437",
+    strict=False,
+)
```

Comment:
As commented by @thom311 in the mentioned BZ, the `arp_validate` is only relevant for `active-backup` bond mode while this test uses `balance-rr`. The test needs to use a different option.

Reply:
I have removed this kind of test case.